### DISCUSSION TITLE
Should osu!mania have barlines? Continuation of #19611

### DISCUSSION THREAD

**Discussion by Redageddon** [ppy/osu#19674] [Polls]
Google poll at: https://www.reddit.com/r/osugame/comments/wkor9h/osumania_barline_poll/

**Comment by peppy** [comment]
I mean what's the point of this? If you don't want them, you can skin them away? We're not removing them nor adding a toggle. If users want to break their game experience, they can skin as always.

**Reply by Redageddon** [reply]
I never asked for a toggle though. Just to add a way for them to be removable. And you haven't said up until just right now that it would be possible to skin them away in the future. You were giving me the assumption that they would be here to stay, and only the way in which they stay may or may not be different.

**Reply by peppy** [reply]
Are they not skinnable in stable? Anything which works there will work here.

**Reply by smoogipoo** [reply]
![image](https://user-images.githubusercontent.com/1329837/183838139-ba06da6d-d2d8-4e3e-9be8-1f27abe158fa.png)

Looks like they are skinnable.

**Reply by ItsCollector** [reply]
I play with ColourBarline all set to 0,0,0,0 so I can't see it, however it does have its use for SV maps in particular it helps judge the speed change faster. Go into your skin and do "ColourBarline: 0,0,0,0" if you want to hide it too.
